apps/web/package.json

```diff
@@ -101,11 +101,11 @@
     "memory-cache": "^0.2.0",
     "micro": "^10.0.1",
     "mime-types": "^2.1.35",
-    "next": "^13.5.6",
+    "next": "^15.1.6",
     "next-auth": "^4.22.1",
     "next-axiom": "^0.17.0",
     "next-collect": "^0.2.1",
-    "next-i18next": "^13.2.2",
+    "next-i18next": "^15.4.2",
```

Comment:
Upgrade doesn't really help with server vs client code errors, but still doing it, why not :D